code/modules/mob/living/basic/ruin_defender/wizard/wizard.dm

```diff
@@ -0,0 +1,91 @@
+/mob/living/basic/wizard
+	name = "Space Wizard"
+	desc = "A wizard is never early. Nor is he late. He arrives exactly at the worst possible moment."
+	icon = 'icons/mob/simple/simple_human.dmi'
+	icon_state = "wizard"
+	icon_living = "wizard"
+	icon_dead = "wizard_dead"
+	mob_biotypes = MOB_ORGANIC|MOB_HUMANOID
+	sentience_type = SENTIENCE_HUMANOID
+	speed = 0
+	maxHealth = 100
+	health = 100
+	melee_damage_lower = 5
+	melee_damage_upper = 5
+	attack_verb_continuous = "punches"
+	attack_verb_simple = "punch"
+	attack_sound = 'sound/weapons/punch1.ogg'
+	combat_mode = TRUE
+	habitable_atmos = list("min_oxy" = 5, "max_oxy" = 0, "min_plas" = 0, "max_plas" = 1, "min_co2" = 0, "max_co2" = 5, "min_n2" = 0, "max_n2" = 0)
+	unsuitable_atmos_damage = 7.5
+	faction = list(ROLE_WIZARD)
+	basic_mob_flags = DEL_ON_DEATH
+	ai_controller = /datum/ai_controller/basic_controller/wizard
+
+	/// A list of possible wizard corpses, and therefore wizard outfits, to select from
+	var/static/list/wizard_outfits = list(
+		/obj/effect/mob_spawn/corpse/human/wizard = 5,
+		/obj/effect/mob_spawn/corpse/human/wizard/red = 3,
+		/obj/effect/mob_spawn/corpse/human/wizard/yellow = 3,
+		/obj/effect/mob_spawn/corpse/human/wizard/black = 3,
+		/obj/effect/mob_spawn/corpse/human/wizard/marisa = 1,
+		//The tape wizard should go here, but its hat doesn't render correctly for some reason.
+	)
+	/// A specified wizard corpse spawner to use. If null, picks from the list above instead.
+	var/selected_outfit
+
+	/// Typepath for the wizard's targeted spell. If null, selects randomly.
+	var/targeted_spell_path
+	/// List of possible targeted spells to pick from
+	var/static/list/targeted_spell_list = list(
+		/datum/action/cooldown/spell/pointed/projectile/fireball/lesser,
+		/datum/action/cooldown/spell/pointed/projectile/lightningbolt,
+		/datum/action/cooldown/spell/pointed/projectile/spell_cards,
```

Comment:
Spell Cards is probably no good considering it's not just fire and forget and is charged based when the others is not

Reply:
That's a good point. I don't really like having only two main spells, but spell cards are really, really weak for AI to use. I've removed it.